tka/aum.go

```diff
@@ -59,10 +59,9 @@ const (
 	//
 	// Only the KeyID optional field may be set.
 	AUMRemoveKey
-	// A DisableNL AUM describes the disablement of TKA.
-	//
-	// Only the DisablementSecret optional field may be set.
-	AUMDisableNL
+	// Formerly DisableNL, This AUM type is unused but kept to
+	// maintain ordering.
+	AUMUnused1
```

Comment:
I feel like maybe either:
- The test inputs that use this ID should be kept around to make sure that existing data continues to process without making things explode, or
- Removing the ID altogether makes sense, and we assume that old data using the old IDs will never be seen.

Right now it seems like if `AUMUnused1` is found, that's not an error, but nothing generates it anymore and it's ignored if seen.

Reply:
Ive gone ahead and removed it altogether, given theres nothing relying on this in the field yet. That did necessitate updating a bunch of tests, but no biggie.